pkg/runctl/run_manager_test.go

```diff
@@ -1782,6 +1825,151 @@ func Test_RunManager_Log_Elasticsearch(t *testing.T) {
 			assert.Assert(t, is.Nil(param))
 		})
 	}
+
+	/**
+	 * Test: If provided indexURL at spec.logging.elasticsearch.indexURL
+	 * does not have correct format test should fail.
+	 */
+	test = "ErrorOnWrongFormattedIndexURL"
+	for _, tc := range []struct {
+		name string
+		URL  string
+	}{
+		{"indexURLWithNoScheme", `"indexURL": "testURL"`},
+		{"indexURLWithWrongFormat2", `"indexURL": "http//testURL"`},
+	} {
+		t.Run(test+"_"+tc.name, func(t *testing.T) {
+			// setup
+			pipelineRunJSON := fmt.Sprintf(fixIndent(`
+				{
+					"apiVersion": "steward.sap.com/v1alpha1",
+					"kind": "PipelineRun",
+					"metadata": {
+						"name": "dummy1",
+						"namespace": "namespace1"
+					},
+					"spec": {
+						"jenkinsFile": {
+							"repoUrl": "dummyRepoUrl",
+							"revision": "dummyRevision",
+							"relativePath": "dummyRelativePath"
+						},
+						"logging": {
+							"elasticsearch": {
+								%s
+							}
+						}
+					}
+				}`),
+				tc.URL,
+			)
+			t.Log("input:", pipelineRunJSON)
+			examinee, runCtx, _ := setupExaminee(t, pipelineRunJSON)
+			// exercise
+			resultError := examinee.createTektonTaskRun(runCtx)
+
+			// VERIFY
+			assert.ErrorContains(t, resultError, "field spec.logging.elasticSearch.indexURL is invalid: scheme not supported")
+		})
+	}
+
+	/**
+	 * Test: If `spec.logging.elasticsearch.indexURL` has an unsupported
+	 * scheme, an error is returned.
+	 */
+	test = "ErrorOnWrongSchemeInIndexURL"
+	for _, tc := range []struct {
+		name string
+		URL  string
+	}{
+		{"indexURLWithIncorrectScheme", `"indexURL": "ftp://testURL"`},
```

Comment:
Add at least one more test case to justify a parametrized test.

Reply:
This test will not work anymore because we removed configured IndexURL from tekton parameters here: https://github.com/SAP/stewardci-core/pull/172#discussion_r526677601
I'll remove the corresponding tests for now and we can add them in the future when we add IndexURL in Tekton params